airflow/providers/jenkins/operators/jenkins_job_trigger.py

```diff
@@ -110,7 +114,9 @@ def __init__(self,
         self.jenkins_connection_id = jenkins_connection_id
         self.max_try_before_job_appears = max_try_before_job_appears
 
-    def build_job(self, jenkins_server):
+    def build_job(
+            self, jenkins_server: Jenkins, parameters: Optional[str] = "",
```

Comment:
Making parameters a local variable here allow us to make the type checking here easier. It also means we are no longer updating an attribute outside of `__init__`.